stdlib/3.4/enum.pyi

```diff
@@ -20,8 +21,6 @@ class Enum:
 class IntEnum(int, Enum):
     value = ...  # type: int
 
-_T = TypeVar('_T')
-
 def unique(enumeration: _T) -> _T: ...
```

Comment:
I thought this might break because of the new bound on `_T`, since this decorator applies to Enum classes, not Enum instances. But I tried it and it seems to work with mypy.